src/components/Navigation.tsx

```diff
@@ -9,16 +9,22 @@ function Navigation(): JSX.Element {
     <nav role="navigation">
       <ul className={styles.navigation}>
         <li className={styles.navigationItem}>
-          <Link to="/">Home</Link>
+          <Link to="/">HOME</Link>
```

Comment:
I think styling this as a Chakra `Link` might be nice - https://chakra-ui.com/docs/navigation/link#usage-with-routing-library

For now, don't worry too much about appearance - we can fix that with theming later

Reply:
As an example, it might look like:
```suggestion
          <Link as={GatsbyLink} to="/">HOME</Link>
```

Where we change the import at the top to
```
import { Link as GatsbyLink } from 'gatsby';
```